compiler/parser.nim

```diff
@@ -770,6 +771,13 @@ proc parseOperators(p: var TParser, headNode: PNode,
 
 proc simpleExprAux(p: var TParser, limit: int, mode: TPrimaryMode): PNode =
   result = primary(p, mode)
+  if p.tok.tokType == tkCurlyDotLe and
+     p.lex.lineNumber == result.info.line and
```

Comment:
The goal of this patch is to enable support for attaching pragmas to call sites. I think that's quite useful ability in general. In this branch, it's used for the "explain" pragma.